### PR TITLE
Avatar selector user schema

### PR DESCRIPTION
**Pull request by AndrewJakubowicz**
## Features

 - Adds avatar thangType id for level and cinematic thangType.
 - Converts vuex `me` store to javascript.
 - Adds hard coded array of avatars.

### CODE REVIEW

File: app/core/store/modules/me.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import _ from 'lodash'
```
Comment (jmif) [inline app/core/store/modules/me.js]: Yay JS

File: app/core/store/modules/me.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      // updates this module, Backbone me, and server
+      const user = _.assign({}, state, updates)
+      return api.users.put(user)
+        .then(() => {
+          me.set(updates) // will also call updateUser
+          return state
+        })
+    },
+
+    setAvatar ({ state, commit }, { levelThangTypeId, cinematicThangTypeId }) {
```
Comment (jmif) [inline app/core/store/modules/me.js]: Maybe `set1fhAvatar`?  I can see a future world where users actually have their own avatars.

File: app/schemas/models/user.coffee [hunk 3]
```diff
@@ -170,6 +170,13 @@ _.extend UserSchema.properties,
           }),
           colorGroups: c.object({ additionalProperties: c.colorConfig() })
         }))
+      avatar: c.object({
+        title: 'Avatar Choice',
```
Comment (jmif) [inline app/schemas/models/user.coffee]: Same here maybe 1FH Avatar Choice?

File: ozaria/site/components/avatar-selector/PageAvatarSelector/index.vue [hunk 4]
```diff
@@ -85,35 +134,29 @@ export default Vue.extend({
...
-              </div>
-
-              <div class="col-xs-4 avatar-item">
-                <div class="avatar1" :class="{selected: selected === 2}" data-avatar="2" @click="handleClick" />
-              </div>
-
-              <div class="col-xs-4 avatar-item">
-                <div class="avatar3" :class="{selected: selected === 3}" data-avatar="3" @click="handleClick" />
+              <!-- TODO: COMPUTED FIRST ROW OF AVATAR. COMPUTED SECOND ROW OF AVATAR. -->
+              <div class="col-xs-4 avatar-item" v-for="({ selectionImg }, index) in topRowAvatars" :key="index">
```
Comment (jmif) [inline ozaria/site/components/avatar-selector/PageAvatarSelector/index.vue]: Could the key be a thang id?  Using an index actually has some edge cases that cause rendering not to occur.  It's best to associate the key with something unique to the entry if possible.

File: ozaria/site/components/avatar-selector/PageAvatarSelector/index.vue [hunk 5]
```diff
@@ -85,35 +134,29 @@ export default Vue.extend({
...
-                <div class="avatar2" :class="{selected: selected === 1}" data-avatar="1" @click="handleClick" />
-              </div>
-
-              <div class="col-xs-4 avatar-item">
-                <div class="avatar1" :class="{selected: selected === 2}" data-avatar="2" @click="handleClick" />
-              </div>
-
-              <div class="col-xs-4 avatar-item">
-                <div class="avatar3" :class="{selected: selected === 3}" data-avatar="3" @click="handleClick" />
+              <!-- TODO: COMPUTED FIRST ROW OF AVATAR. COMPUTED SECOND ROW OF AVATAR. -->
```
Comment (jmif) [inline ozaria/site/components/avatar-selector/PageAvatarSelector/index.vue]: This might be easier if we use flexbox with width 33% on the flex items.

Comment (jmif) [reply ozaria/site/components/avatar-selector/PageAvatarSelector/index.vue]: Ah its already done above, ok.  Probably don't need this TODO then.

Comment (AndrewJakubowicz) [reply ozaria/site/components/avatar-selector/PageAvatarSelector/index.vue]: oops good catch!

**Pull request merged**